packages/@tinacms/graphql/src/database/level.ts

```diff
@@ -48,3 +48,54 @@ export const SUBLEVEL_OPTIONS: AbstractSublevelOptions<
   separator: INDEX_KEY_FIELD_SEPARATOR,
   valueEncoding: 'json',
 }
+
+export class LevelWrapper {
+  private level: Level
+  constructor(level: Level) {
+    this.level = level
+  }
+}
+
+const LevelProxyHandler = {
+  get: function (target, property) {
+    if (!target[property]) {
+      throw new Error(`The property, ${property.toString()}, doesn't exist`)
+    }
+    if (typeof target[property] !== 'function') {
+      throw new Error(`The property, ${property.toString()}, is not a function`)
+    }
+
+    if (property === 'get') {
+      return async (...args) => {
+        let result: any
+        try {
+          // eslint-disable-next-line prefer-spread
+          result = await target[property].apply(target, args)
+        } catch (e: any) {
+          if (e.code !== 'LEVEL_NOT_FOUND') {
+            throw e
+          }
+        }
+        return result
+      }
+    } else if (property === 'sublevel') {
+      // wrap sublevel in a proxy to intercept get
+      return (...args) => {
+        // eslint-disable-next-line prefer-spread
+        return new Proxy(
+          target[property].apply(target, args),
```

Comment:
@kldavis4 might have to move this comment down one line to skip the eslint check. `// eslint-disable-next-line prefer-spread`